src/openrct2/ride/Ride.h

```diff
@@ -136,6 +136,22 @@ struct rct_ride_entry
 
 #pragma pack(pop)
 
+struct RideStation
+{
+    LocationXY8 Start;
+    uint8_t Height;
+    uint8_t Length;
+    uint8_t Depart;
+    uint8_t TrainAtStation;
+    TileCoordsXYZD Entrance;
+    TileCoordsXYZD Exit;
+    int32_t SegmentLength;
+    uint16_t SegmentTime;
```

Comment:
What do SegmentLength and SegmentTime mean?
I don't think they're very clear terms - at the very least, they need a comment.